containers/runner/launch

```diff
@@ -61,8 +65,19 @@ fi
 mkdir -p data/images
 mkdir -p -m 777 data/logs
 if ! [ -e data/images/boot.iso ]; then
-    echo "INFO: data/images/boot.iso does not exist, downloading current Fedora Rawhide Server image..."
-    curl -L https://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Server/x86_64/os/images/boot.iso --output data/images/boot.iso
+    if [ -n "${DAILY_COPR_TOKEN:-}" ]; then
+        echo "INFO: data/images/boot.iso does not exist, downloading daily rawhide+COPR iso..."
+        CURL="curl -u token:$(cat $DAILY_COPR_TOKEN) --show-error --fail"
```

Comment:
Hmm I wonder why we need token to download test artifacts which are present publicly...

Reply:
They aren't -- if you log out of GitHub (or use private mode), you can't see the artifacts. You just get a note like "log in to see the artifacts". Curiously you don't even need to have particular permissions for *this* project -- I tried this with a cockpituous token just for fun, and that worked. To me it mostly just feels like a mild prevention of abusing GitHub as a generic file upload/download place, so that you can't do anonymous illegal file stores/activities.